Homework_3/task_1.py

```diff
@@ -0,0 +1,19 @@
+# 1
+numbers = [1, 2, 3, 4, 5, 6, 7, 8]
+even_index = []
+even_index_numbers = []
+odd_index = []
+odd_index_numbers = []
```

Comment:
to complicated. you actually need only 2 additional lists. and tan in your loop just put elements as tuplea. 

some_list.append((index, value))